kani-driver/src/assess/mod.rs

```diff
@@ -8,13 +8,22 @@ use crate::metadata::merge_kani_metadata;
 use crate::project;
 use crate::session::KaniSession;
 
+pub use self::args::AssessArgs;
+
+mod args;
+mod metadata;
+mod scan;
 mod table_builder;
 mod table_failure_reasons;
 mod table_promising_tests;
 mod table_unsupported_features;
 
-pub(crate) fn cargokani_assess_main(mut session: KaniSession) -> Result<()> {
-    // fix some settings
+pub(crate) fn cargokani_assess_main(mut session: KaniSession, args: AssessArgs) -> Result<()> {
+    if let Some(args::AssessSubcommand::Scan(args)) = &args.command {
+        return scan::assess_scan_main(session, args);
+    }
+
+    // fix (as in "make unchanging/unchangable") some settings
     session.args.unwind = Some(1);
```

Comment:
Will we allow this to be configurable?

Reply:
The way args are currently set here is a temporary hack until we can refactor how arguments are set in kani-driver entirely.

In the longer-run, this particular one won't be a thing at all: we'd switch to timeout instead of `unwind(1)` to ensure termination, and there's probably be a configurable assess timeout option.